### PR TITLE
3d map example

### PR DESCRIPTION
**Pull request by jmacura**
Note that this PR actually adds two new examples in 'angular9' branch:
- 3D map
- WMTS
Because of its previous rebase on master.

### CODE REVIEW

**Comment by raitisbe** [comment]
Take a look at https://github.com/hslayers/project-agro-climatic

**Comment by jmacura** [comment]
> 
> 
> Take a look at https://github.com/hslayers/project-agro-climatic it uses the hscesium which is rewriten to ng9 [hslayers/hslayers-ng#862](https://github.com/hslayers/hslayers-ng/pull/862)

Thanks for the tip. I did indeed. But I am still struggling with some 'nosniff' error, which is caused (I believe) by improperly set paths to Cesium. If you could take a look at the code, @raitisbe , I would be very pleased.

File: 3d-map/webpack.common.js [hunk 1]
```diff
@@ -44,7 +47,18 @@ module.exports = {
       inject: false,
       // favicon: 'assets/img/favicon.ico'
     }),
+    new CopywebpackPlugin({ patterns: [{ from: path.resolve(path.join(cesiumSource, cesiumWorkers)), to: 'Workers' }] }),
```
Comment (jmacura) [inline 3d-map/webpack.common.js:50]: Oh, that's the trick! 😮

Comment (raitisbe) [reply 3d-map/webpack.common.js:50]: Its just for the assets loaded during runtime, but actualy I was struggling to get it to work until I changed this line https://github.com/hslayers/examples/pull/4/files/5203e5f0a961f846f3e0cfc74bc602e51ebdf140..d8d5ee13285ed13b2e073218aede8d322088950d#diff-5cd1e29f182718e6bd65f2c37e4b3b79R36

**Comment by jmacura** [comment]
Remains to solve:
* Cesium and OL view are not synchronised
* Vector layer won't display on start-up

Hence I keep this PR in a draft state until the above is resolved.

**Comment by raitisbe** [comment]
I created a new example named "cesium" in ng10 branch which uses the latest haslayers. Dont know what to do with this PR. Should we scrap it or upgrade?

**Comment by jmacura** [comment]
> 
> 
> @jmacura I created a new example named "cesium" in ng10 branch which uses the latest haslayers. Dont know what to do with this PR. Should we scrap it or upgrade?

Great news! I gonna cherry-pick what's worth it and close this PR.

**Pull request closed without merging**